cfme/fixtures/configure_auth_mode.py

```diff
@@ -88,3 +96,8 @@ def configure_external_auth_ipa_module(request):
 @pytest.fixture(scope="class")
 def configure_external_auth_ipa_class(request):
     _configure_external_auth_ipa(request)
+
+
+@pytest.fixture(scope="class")
+def configure_openldap_external_auth_mode(request):
+    _configure_openldap_external_auth(request)
```

Comment:
We shouldn't ever ```call``` a fixture.
```python
@pytest.fixture(scope="class")
def configure_openldap_external_auth_mode(_configure_openldap_external_auth):
    pass
```
That should be how the fixture is referenced, but obviously that looks a little silly.

Why split it up like this? Is it to get around a scoping issue?